packages/terra-icon/src/terra-dev-site/test/icon/IconAll.test.jsx

```diff
@@ -46,6 +46,7 @@ const IconAll = () => (
     <Icon.IconChevronUp className={cx(['icon'])} />
     <Icon.IconCircleLinesSymbol className={cx(['icon'])} />
     <Icon.IconCircleSymbol className={cx(['icon'])} />
+    <Icon.IconCircleSymbolLight className={cx(['icon'])} />
```

Comment:
Were some of these just missing on our test page?

Reply:
Yes. There were quite a few icons missing, so I got the test page back up to date.